### PR TITLE
Proposal for meeting 2019-10-09

### PR DESCRIPTION
**Pull request by j-fu**
Hi,
as discussed at the last meeting, here is my proposal for my contribution in October.

Jürgen

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -31,9 +31,11 @@ <h3>Next meeting</h3>
       <p>
         <b>Topics:</b>
         <p>
-            <a href="https://github.com/BeastyBlacksmith">Simon</a> will explain how bacteria find their center and shows how to match experiments with julia.
+          <a href="https://github.com/j-fu">Jürgen</a> will give an introduction to the solution of
+          partial differential equations using the finite volume method and discuss his package
+          <a href="https://github.com/j-fu/VoronoiFVM.jl">VoronoiFVM.jl</a>.
         </p>
-        <b>Date/time:</b> Tuesday, September 10, 2019, 18:30</br>
+        <b>Date/time:</b> Tuesday, October 9, 2019, 18:30</br>
```
Comment (rschwarz) [inline index.html]: Shoul be *October 8*, I guess.

Comment (j-fu) [reply index.html]: Indeed... Please feel free to change this..

Jürgen

**Comment by daveh19** [comment]
@SimonDanisch are you hosting next week? Your message on Slack was a bit vague.

**Comment by daveh19** [comment]
That's ok, I was just going through projects and slack channels for my other project and noticed the mis-match in information here.

When will you be able to host again?


--- Let's discuss this on Slack, it looks like everybody is there.

**Comment by rschwarz** [comment]
@SimonDanisch: Are we going to have a meeting in November, on the 12th?

@j-fu: Are you still available to present?

I can also do a 10 minute presentation about some (refined) triangulations and finding minimum spanning trees in there, featuring Makie-based 2D graphics :-)

**Comment by j-fu** [comment]
@rschwarz   no deadline for the 13th, so I would be available on the 12t

**Comment by rschwarz** [comment]
I resolved the merge conflict, but I'm not sure whether we will have a "regular" meeting in November, with two other events planned for this month already.

**Comment by j-fu** [comment]
Ok no problem  - shifting to next month would be ok for me as well.

**Comment by BeastyBlacksmith** [comment]
I changed the date and added WIAS to locations, would be nice if someone reviews

**Comment by rschwarz** [comment]
Are we meeting at WIAS also in December?
Otherwise, I would not list multiple locations for the "regular" meeting.

**Comment by BeastyBlacksmith** [comment]
Yeah, we decided to regularly rotate the location.

**Comment by rschwarz** [comment]
OK, sounds good to me.

**Comment by BeastyBlacksmith** [comment]
Corrected the location. Ready to merge.

**Pull request merged**